data/sql/updates/pending_db_world/rev_1661826488559382300.sql

```diff
@@ -0,0 +1,38 @@
+DELETE FROM `creature_template_locale` WHERE `entry`=34830 AND `locale` IN ('esMX', 'esES');
+INSERT INTO `creature_template_locale` (`entry`, `locale`, `Name`, `FemaleName`, `Title`, `VerifiedBuild`) VALUES
+(34830, 'esMX', 'Trol desafiante', '', '', 18019),
+(34830, 'esES', 'Trol desafiante', '', '', 18019);
```

Comment:
This verified build does not correspond with cataclysm, is for pandaria, so please check the data.